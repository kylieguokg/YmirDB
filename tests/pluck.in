pluck no
set a 1 2 3
set b 4 5 6
pluck a 0
pluck a 2
pluck b 2
get a
get b
push a b
list ENTRIES
pluck a 1
list ENTRIES
bye
