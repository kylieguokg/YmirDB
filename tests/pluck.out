> no such key

> ok

> ok

> index out of range

> 2

> 5

> [1 3]

> [4 6]

> ok

> b [4 6]
a [b 1 3]

> b

> b [4 6]
a [1 3]

> bye
